rflx/integration.py

```diff
@@ -0,0 +1,174 @@
+from pathlib import Path
+from typing import Dict, Optional
+
+from pydantic import BaseModel, Field, ValidationError
+from pydantic.types import ConstrainedInt
+
+from rflx.error import Location, RecordFluxError, Severity, Subsystem
+from rflx.identifier import ID
+from rflx.model import Model
+from rflx.model.session import Session
+
+
+class IntSize(ConstrainedInt):
+    gt = 0
+
+
+class SessionSize(BaseModel):
+    default: Optional[IntSize] = Field(alias="Default")
+    global_: Optional[Dict[str, IntSize]] = Field(alias="Global")
+    local_: Optional[Dict[str, Dict[str, IntSize]]] = Field(alias="Local")
```

Comment:
Extra attributes should be forbidden (e.g., to detect typos), cf. https://pydantic-docs.helpmanual.io/usage/model_config/.

Reply:
Fixed in 08de02fd54a8b67246fe527c391c005ebf7a98bc